src/components/CheckboxGroup/index.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { uniqueId } from '../../libs/utils';
+import RenderIf from '../RenderIf';
+import RequiredAsterisk from '../Input/requiredAsterisk';
+import Checkbox from './checkbox';
+import './styles.css';
+
+export default class CheckboxGroup extends Component {
+    constructor(props) {
+        super(props);
+        this.handleOnChange = this.handleOnChange.bind(this);
+    }
+
+    getCheckboxValues() {
+        const { options } = this.props;
+        return options.map(option => (
+            <Checkbox
+                {...option}
+                isSelected={this.checkIfSelected(option)}
+                onChange={this.handleOnChange}
+                key={uniqueId('checkbox')} />
+        ));
+    }
+
+    getCheckboxContainerClassNames() {
+        const { error } = this.props;
+        return classnames('rainbow-checkbox-group-container', { 'rainbow-has-error': !!error });
```

Comment:
ranibow-has-error class does not follow the format:
rainbow-component-name-modifiers